### DISCUSSION TITLE
does not see the metamask connector

### DISCUSSION THREAD

**Discussion by onufrienkoPr** [wevm/wagmi#2189] [Q&A] [answered]
good afternoon, what could be the problem if I have the metamask extension installed, I don’t see the connector until I refresh the page, it will also happen if I follow the link https://wagmi.sh/examples/connect-wallet and the button is not displayed until I refresh page?

**Comment by tmm** [comment]
Do you have any other browser wallet extensions installed (e.g. Coinbase, Brave, Trust)?

**Reply by onufrienkoPr** [reply]
Yes

**Reply by onufrienkoPr** [reply, marked as answer]
I deleted coinbase - aaaaaaand metamask was displayed immediately wtf

**Reply by onufrienkoPr** [reply]
apparently the browser does not understand the metamask connector when other wallet extensions are installed

**Reply by tmm** [reply]
Yup, exactly! Injected providers tend to fight over `window.ethereum`. Hopefully, this will get addressed in a future EIP.

**Reply by johnpaulcas** [reply]
hi @tmm is this issue resolve? we are having the same issue when metamask and other wallets are installed, it seems can't detect the metamask on the browser 

![Screenshot 2023-07-05 at 2 29 03 PM](https://github.com/wagmi-dev/wagmi/assets/44362950/aba29739-b6e1-4b9f-9f90-85af0dd2ea4a)
